website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -701,7 +733,7 @@ curl \
   https://app.terraform.io/api/v2/policy-sets/polset-3yVQZvHzf5j3WRJ1/relationships/policies
 ```
 
-## Attach a Policy Set to projects
+## Attach a Policy Set to Projects
```

Comment:
@rkoron007 fixed Capitalization of the titles to make them consistent.

Reply:
Thank you so much for putting in the effort here.  We've recently [updated the style guide](https://github.com/hashicorp/engineering-docs/blob/main/writing/style-guide.md#titles-and-headers) to specify that headers should use sentence casing. It's a not a huge priority, but it you are willing to make the change across the page to ensure all headings use sentence casing instead that would be **amazing**! No problem if you'd rather not though, just let me know and I can handle it later!

We also don't capitalize feature names, just _product_ names (so we capitalize Terraform Cloud, but not projects which are a _feature_ of Terraform Cloud). So we'd ideally update this to:
```suggestion
## Attach a policy set to projects
```

Reply:
I have fixed it in all places on this page.